src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -106,7 +105,7 @@ function BaseOnboardingAccounting({shouldUseNativeStyles, route}: BaseOnboarding
                         StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR),
                         styles.mr3,
                         onboardingIsMediumOrLargerScreenWidth ? styles.ml3 : styles.ml0,
-                        styles.onboardingIconWrapper,
+                        {padding: 10},
```

Comment:
Inline styles are discouraged. Let's make a style for this icon

Reply:
Updated.